src/Widgets/HeaderBar.vala

```diff
@@ -19,12 +19,9 @@
 * Authored by: Alessandro "Alecaddd" Castellani <castellani.ale@gmail.com>
 */
 
-public class Akira.Widgets.HeaderBar : Gtk.Box {
```

Comment:
In general, the use of Gtk.Box is discouraged. It's an old widget and it adheres to old Gtk2 ways of doing things like specifying padding when packing widgets

Reply:
Sounds good.
In case I need a similar solution, is there something I could use instead of the `Gtk.Box` ?

Reply:
The recommended replacement is Gtk.Grid. You can use it like a box, if you want. It has an orientation properly and you can use the `add` method with it instead of `attach`